python-threatexchange/threatexchange/TE.py

```diff
@@ -13,35 +13,28 @@
 # General Python dependencies
 import os
 import re
-import requests
+import threading
+import urllib.parse
 
+import requests
 from requests.adapters import HTTPAdapter
 from requests.packages.urllib3.util.retry import Retry
 
-import urllib.parse
-
-
-DEFAULT_TIMEOUT = 5  # seconds
-
 
 class TimeoutHTTPAdapter(HTTPAdapter):
     """
     Plug into requests to get a well-behaved session that does not wait for eternity.
     H/T: https://findwork.dev/blog/advanced-usage-python-requests-timeouts-retries-hooks/#setting-default-timeouts
     """
 
-    def __init__(self, *args, **kwargs):
-        self.timeout = DEFAULT_TIMEOUT
-        if "timeout" in kwargs:
-            self.timeout = kwargs["timeout"]
-            del kwargs["timeout"]
+    def __init__(self, *args, timeout=5, **kwargs):
```

Comment:
nit/ignorable: this 5 and 60 below are kind of magic numbers but imo fine until it's not a single point of maintenance

Reply:
Yea, I prefer to inline unless it needs to be referenced. Otherwise you have to hunt to the place in the code where it used to tell what it means anyways.